ext/openssl/openssl.stub.php

```diff
@@ -105,39 +105,39 @@ function openssl_pkey_get_details(OpenSSLAsymmetricKey $key): array|false {}
 
 function openssl_pbkdf2(string $passphrase, string $salt, int $key_length, int $iterations, string $digest_algorithm = "sha1"): string|false {}
 
-function openssl_pkcs7_verify(string $filename, int $flags, ?string $output_filename = null, array $ca_info = [], ?string $untrusted_certificates_filename = null, ?string $content = null, ?string $pk7_filename = null): bool|int {}
+function openssl_pkcs7_verify(string $filename, int $flags, ?string $signers_certificates_filename = null, array $ca_info = [], ?string $untrusted_certificates_filename = null, ?string $content = null, ?string $pkcs7_output_filename = null): bool|int {}
 
 /** @param OpenSSLCertificate|array|string $certificate */
-function openssl_pkcs7_encrypt(string $filename, string $output_filename, $certificate, ?array $headers, int $flags = 0, int $cipher_algorithm = OPENSSL_CIPHER_RC2_40): bool {}
+function openssl_pkcs7_encrypt(string $input_filename, string $pkcs7_output_filename, $certificate, ?array $headers, int $flags = 0, int $cipher_algorithm = OPENSSL_CIPHER_RC2_40): bool {}
```

Comment:
@bukka The PR is not yet complete because I wonder if we really need the `pkcs7` and `cms` prefixes in case of the `$input_filename` and `$output_filename` parameters? It's not a very big deal for me, but I'd like to make sure :)

Reply:
I guess it might not be actually necessary to prefix it. If it's `input_filename` and `output_filename`, then it can be sort of clear from the context. I think such detail should be more in the docs rather than the parameter name.